#Service to stop logging the state of the system
---
int64 ret        # Set to 1 (success) or 0 (error)
string msg       # Error description
